WORKSPACE

```diff
@@ -89,7 +89,7 @@ tf_workspace()
 load("@bazel_skylib//lib:versions.bzl", "versions")
 # Keep this version in sync with the BAZEL environment variable defined
 # in our .travis.yml config.
-versions.check(minimum_bazel_version = "2.1.0")
+versions.check(minimum_bazel_version = "3.0.0")
```

Comment:
> upgrades bazel from 2.1.0 to 3.7.0 with minimum requirement of 3.0.0.

IIRC, in the past we wanted to keep our CI Bazel version in sync with
our minimum stable Bazel version, because otherwise nothing checks that
we’re actually compatible at the version that we claim to be. Are we
intentionally reverting that?